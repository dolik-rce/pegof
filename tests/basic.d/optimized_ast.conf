optimize all
ast
input basic.d/basic.peg
