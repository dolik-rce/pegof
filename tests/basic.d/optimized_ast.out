GRAMMAR:
  RULE:
    RULE_NAME: "main" line=1, col=1
    ALTERNATION:
      STRING: "AB" (optimized)
      STRING: "C" line=4, col=7
